.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+
+version: 2
+enable-beta-ecosystems: true
+updates:
+ - package-ecosystem: "devcontainers"
+   directory: "/"
+   schedule:
+     interval: weekly
+ - package-ecosystem: "docker"
+   directory: "/"
+   schedule:
+     interval: weekly
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 5

(indentation)